build.py

```diff
@@ -1062,10 +1081,11 @@ def container_build(backends, images):
         log('image "{}": "{}"'.format(parts[0], parts[1]))
         images[parts[0]] = parts[1]
 
-    # If --container-build is specified then we perform the actual
+    # If --no-container-build or jetson_build is not specified then we perform the actual
```

Comment:
Your jetson build should just explicitly add --no-container-build

Reply:
Done